hpy/devel/src/runtime/argparse.c

```diff
@@ -78,6 +78,11 @@
  *     `Truth Value Testing <https://docs.python.org/3/library/stdtypes.html#truth>`_
  *     for more information about how Python tests values for truth.
  *
+ * ``s (unicode) [const char*]``
```

Comment:
I guess `s` should go in its own new section, not under `Miscellaneous`. The CPython documentation uses the heading `Strings and buffers` which probably makes sense since they share many common considerations (e.g. the lifetime and deallocation of the returned memory).

Reply:
We should also mention precisely what the returned buffer contains (i.e. UTF-8 bytes with a NUL at the end). I would recommend actually copying the CPython documentation for `s` and then adjusting it as necessary.

Reply:
Fixed